INSTALLATION.md

```diff
@@ -0,0 +1,142 @@
+# Intstallation Guide
+
+## Prerequisites
+
+### Check if OpenStack system is functional
+* Build, destry VMs, etc.
+* Deploy Managed K8, test it
+
+### Steps to prepare environment (on each host)
+
+1. Verify VLan is connected, e.g. by pinging from one infra host to the other
+2. Setup Bridge and v-eth pair (see ![Alt](rpc-octavia-network-diagram.png "RPC Octavia Network Diagram"))
+
+```bash
```

Comment:
Should probably add some notes on what this script is (representative) and that `VLAN_ID` and `address` might need to be adjusted.